src/mockServer/route.ts

```diff
@@ -0,0 +1,11 @@
+import qs from 'qs';
+
+import { UUID } from '@graasp/sdk';
+
+export const ITEMS_ROUTE = 'items';
+export const buildGetItemRoute = (id: UUID): string => `${ITEMS_ROUTE}/${id}`;
+export const buildGetChildrenRoute = (id: UUID, ordered: boolean): string =>
+  `${ITEMS_ROUTE}/${id}/children${qs.stringify(
+    { ordered },
+    { addQueryPrefix: true },
+  )}`;
```

Comment:
You can get these from query-client from API_ROUTES I think. If you can't then just strings as the other endpoints for consistency.